runtime/src/main/java/org/corfudb/protocols/logprotocol/SMRRecordLocator.java

```diff
@@ -0,0 +1,54 @@
+package org.corfudb.protocols.logprotocol;
+
+import io.netty.buffer.ByteBuf;
+import lombok.*;
+import org.corfudb.util.serializer.ICorfuSerializable;
+
+import java.util.UUID;
+
+
+@ToString
+@AllArgsConstructor
+@NoArgsConstructor
+@EqualsAndHashCode
+@Getter
+@Setter
+public class SMRRecordLocator implements Comparable<SMRRecordLocator>, ICorfuSerializable {
+
+    public static SMRRecordLocator NONE = new SMRRecordLocator(-1L, UUID.randomUUID(), 0);
```

Comment:
public final
Also, you can use null instead of UUID.randomUUID()

Reply:
done